### ISSUE TITLE
GPT2 seq_len invalid config

### ISSUE THREAD

**Issue by samir-souza** [aws-neuron/transformers-neuronx#26]
in the file transformers-neuronx/blob/main/src/transformers_neuronx/gpt2/model.py
all variations of the model are configuring **max seq length** using a wrong parameter. The code bellow is replicated in many part of this file and it correctly tests **n_positions** as the input config for max len, but uses **n_ctx** to get the value. So, please, change this to the correct config var **n_positions** instead.

 ```
sequence_length = kwargs.get("n_positions", None)
 if sequence_length:
            max_allowed_sequence_length = config.n_ctx
            if sequence_length > max_allowed_sequence_length:
                raise ValueError(f"Sequence length ({sequence_length}) cannot be larger than position embedding's context size ({max_allowed_sequence_length})!")
```

**Comment by micwade-aws** [comment]
Thanks for the heads up - will get this fixed.

**Comment by hannanjgaws** [comment]
This issue was fixed as of the `0.7.x` `transformers-neuronx` release. Thanks again for reporting this issue! I'll close this ticket.
